### ISSUE TITLE
SPFx Teams Tab not visible on Android App

### ISSUE THREAD

**Issue by mateusz-chrzastek** [SharePoint/sp-dev-docs#3928]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

SPFx WebPart Teams Tab is visible and accessable on mobile.

#### Observed Behavior

Teams Tab is not visible on the Microsoft Teams App on Android (Version: 1416/1.0.0.2019042206, Calling Version: 2019.11.01.1)

#### Steps to Reproduce

Add a Tab from a Custom App to MS Teams that has a configurableTab which is a WebPart in a SP AppGallery.

Basically this guide: https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/using-web-part-as-ms-teams-tab

**Comment by patmill** [comment]
IANATD (I am not a Teams dev), however...  Support for tabs in the Teams mobile apps is still not a released feature (Tabs created through the traditional teams SDK won't appear either).  I believe there is a hard-coded list of tabs that will show, which is why it looks like it is somewhat supported.  When SPFx based tabs are supported, your tab should automatically show up.  I believe the ETA for this.  Unfortunately, I don't have a timeline for this feature.
